### PR TITLE
Documentation for symfony-cmf/routing-bundle#363

### PR DESCRIPTION
**Pull request by ahilles107**
Documentation for https://github.com/symfony-cmf/routing-bundle/pull/363

### CODE REVIEW

File: bundles/routing/configuration.rst [hunk 1]
```diff
@@ -469,6 +469,13 @@ If ``true``, the ORM is included in the service container.
 
 The name of the Doctrine Manager to use.
 
+``route_class``
+****************
+
+**type**: ``string`` **default**: ``'Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Orm\Route'``
+
+Class used by bundle to work with routes.
```
Comment (dbu) [inline bundles/routing/configuration.rst]: i would say "The route class to use in your application."

Comment (ahilles107) [inline bundles/routing/configuration.rst]: done

**Comment by dbu** [comment]
sorry, one more thing that i just realized: can you please add a line to say this was added in version 2?

after the explanation, add

```
versionadded:: 2.0
    The ``route_class`` parameter has been added in version 2.0.
```

**Comment by dbu** [comment]
thanks you! i will merge as soon as the code has been merged.

**Pull request merged**